CMakeLists.txt

```
@@ -149,14 +169,16 @@ target_link_libraries(idaklu PRIVATE ${SUNDIALS_LIBRARIES} casadi)
 # link suitesparse
 # if using vcpkg, use config mode to
 # find suitesparse. Otherwise, use FindSuiteSparse module
+set(SUITESPARSE_ROOT ${CUSTOM_INSTALL_DIR})
+message("SUITESPARSE_ROOT: ${SUITESPARSE_ROOT}")
 if(DEFINED VCPKG_ROOT_DIR)
-  find_package(SuiteSparse CONFIG REQUIRED)
+  find_package(SUITESPARSE CONFIG REQUIRED)
```

```suggestion
  find_package(SuiteSparse CONFIG REQUIRED)
```

We should be good to go back to the original naming convention